instruqt-tracks/sentinel-for-terraform-v4/10-extra-credit/assignment.md

```diff
@@ -0,0 +1,123 @@
+---
+slug: extra-credit
+id: evcxnlktzbc8
```

Comment:
Let's clear the `id:` fields out of `assignment.md` while we're moving to v3 of the track config